integration-tests/environments/react-native-test-app/package.json

```diff
@@ -97,6 +101,16 @@
     },
     "pod-install:ci": {
       "command": "pod-install",
+      "clean": "if-file-deleted",
+      "files": [
+        "ios/Podfile",
+        "../../../packages/realm/RealmJS.podspec"
+      ],
+      "output": [
+        "ios/Pods",
+        "ios/Podfile.lock",
+        "ios/RealmTests.xcworkspace"
+      ],
```

Comment:
No need for dependencies on other scripts when testing on CI, since that's already handled by other jobs uploading artifacts.